Packs/Netskope/Integrations/NetskopeAPIv2/NetskopeAPIv2_test.py

```diff
@@ -0,0 +1,524 @@
+
+
+def test_update_url_list(client, requests_mock):
+    """
+    Scenario: Update the URL List with the values provided.
+    Given:
+        - User has provided valid credentials.
+    When:
+        - netskope-url-list-update command is called.
+    Then:
+        - Ensure number of items is correct.
+        - Ensure outputs prefix is correct.
+        - Ensure a sample value from the API matches what is generated in the context.
+    """
+
+    from NetskopeAPIv2 import update_url_list_command
+
+    mock_response = util_load_json("url_list")
+
+    requests_mock.put(f"{SERVER_URL}api/v2/policy/urllist/1", json=mock_response)
+
+    result = update_url_list_command(
+        client,
+        {
+            "url_list_id": "1",
+            "name": "new_url_list",
+            "urls": ["www.google.com"],
+            "list_type": "Regex",
+            "is_overwrite": True,
+            "deploy": False,
+        },
+    )
+
+    assert result.outputs_prefix == "Netskope.URLList"
+    assert result.outputs[0]["name"] == "new_url_list"
+    assert result.outputs[0]["urls"][0] == "www.google.com"
+
+
+def test_create_url_list(client, requests_mock):
+    """
+    Scenario: create the URL List with the values provided.
+    Given:
+        - User has provided valid credentials.
+    When:
+        - netskope-url-list-create command is called.
+    Then:
+        - Ensure number of items is correct.
+        - Ensure outputs prefix is correct.
+        - Ensure a sample value from the API matches what is generated in the context.
+    """
+
+    from NetskopeAPIv2 import create_url_list_command
+
+    mock_response = util_load_json("url_list")
+
+    requests_mock.post(f"{SERVER_URL}api/v2/policy/urllist", json=mock_response)
+
+    result = create_url_list_command(
+        client,
+        {
+            "name": "new_url_list",
+            "urls": ["www.google.com"],
+            "list_type": "Regex",
+            "deploy": False,
+        },
+    )
+
+    assert result.outputs_prefix == "Netskope.URLList"
+    assert result.outputs[0]["name"] == "new_url_list"
+    assert result.outputs[0]["urls"][0] == "www.google.com"
+
+
+def test_delete_url_list(client, requests_mock):
+    """
+    Scenario: delete the URL List with the values provided.
+    Given:
+        - User has provided valid credentials.
+    When:
+        - netskope-url-list-delete command is called.
+    Then:
+        - Ensure number of items is correct.
+        - Ensure outputs prefix is correct.
+        - Ensure a sample value from the API matches what is generated in the context.
+    """
+
+    from NetskopeAPIv2 import delete_url_list_command
+
+    mock_response = util_load_json("url_list")
+
+    requests_mock.delete(f"{SERVER_URL}api/v2/policy/urllist/1", json=mock_response)
+
+    result = delete_url_list_command(client, {"url_list_id": "1", "deploy": False})
+
+    assert result.outputs_prefix == "Netskope.URLList"
```

Comment:
please check the output as well, the `outputs_prefix` is not enough.

Reply:
added another assert for the 'readable_output'.